dev_guide/app_tutorials/maven_tutorial.adoc

```diff
@@ -86,29 +87,32 @@ $ oc set probe dc/nexus \
 ====
 If you do not want persistent storage, continue to
 xref:nexus-connecting-to-nexus[Connecting to Nexus]. However, your cached
-dependencies will be lost if the pod is restarted for any reason.
+dependencies and any configuration customization will be lost if the pod is
+restarted for any reason.
 ====
 
-Create a persistent volume claim (PVC) for Nexus, so that your dependencies
-will not be lost when the pod running the server terminates. If you do not have
-administrator access on your cluster, ask your system administrator to create a
-Read/Write Persistent Volume for you.  Otherwise, read
-xref:../volumes.adoc#dev-guide-volumes[Managing Volumes in {product-title}] for instructions on
-creating a persistent volume.
+Create a persistent volume claim (PVC) for Nexus, so that your dependencies are
+not lost when the pod running the server terminates. PVCs require available
+persistent volumes (PV) in your cluster. If there are no PVs available in your
+cluster and you do not have administrator access on your cluster, ask your
+system administrator to create a Read/Write Persistent Volume for you.
+Otherwise, see
```

Comment:
This paragraph has too much occurrences of `your` word. Maybe *you* can improve it? :-)